app/pages/search/SearchPage.spec.js

```diff
@@ -55,92 +55,43 @@ describe('pages/search/SearchPage', () => {
       expect(resourceMap.prop('showMap')).to.equal(true);
     });
 
-    describe('when no search is done', () => {
-      const extraProps = {
-        isFetchingSearchResults: false,
-        searchDone: false,
-      };
-
-      it('renders SearchControls with correct props', () => {
-        const searchControls = getWrapper(extraProps).find(SearchControls);
-
-        expect(searchControls.length).to.equal(1);
-        expect(searchControls.props().location).to.deep.equal(defaultProps.location);
-        expect(searchControls.props().params).to.deep.equal(defaultProps.params);
-        expect(typeof searchControls.props().scrollToSearchResults).to.equal('function');
-      });
-
-      it('does not render SearchResults', () => {
-        const searchResults = getWrapper(extraProps).find(SearchResults);
-
-        expect(searchResults.length).to.equal(0);
-      });
-
-      it('renders help text', () => {
-        const helpText = getWrapper(extraProps).find('.help-text');
-
-        expect(helpText.length).to.equal(1);
-      });
+    it('renders SearchControls with correct props', () => {
+      const searchControls = getWrapper().find(SearchControls);
+      expect(searchControls.length).to.equal(1);
+      expect(searchControls.props().location).to.deep.equal(defaultProps.location);
+      expect(searchControls.props().params).to.deep.equal(defaultProps.params);
+      expect(typeof searchControls.props().scrollToSearchResults).to.equal('function');
```

Comment:
This could have been:

expect(searchControls.props().scrollToSearchResults).to.be.a('function');